### DISCUSSION TITLE
VONR-Kamailio IMS

### DISCUSSION THREAD

**Discussion by VAyushgupta** [5G-MAG/rt-5gms-application-function#107] [General]
Hi,

I have some queries regarding the AF function:
1.As we are getting the issue of kamailio IMS does not support the N5 interface in order to provision bearers for VoNR call.will it help to resolve the issue?
2.There is no af functionality has been provided by Open5Gs in code. It will help to provide those functionality.If i am wrong please correct me for this.

**Comment by dsilhavy** [comment]
1. Can you please further elaborate what you mean? What is the concrete question?
2. The current version of Open5GS does not provide an AF. We implemented an AF that exercises reference point N5 on the PCF of any 5G core. We used Open5GS service-based library as the foundation for our AF.

**Comment by VAyushgupta** [comment]
Thank you for the response.
for the first point I have the reference which i got as you can see in ss:

![Screenshot from 2023-12-11 10-00-29](https://github.com/5G-MAG/rt-5gms-application-function/assets/128386925/e4c8857d-c558-4709-8fa5-4340d3f81e16)

![Screenshot from 2023-12-06 13-07-54](https://github.com/5G-MAG/rt-5gms-application-function/assets/128386925/342a1100-15b0-4c63-8fc7-779b2934e4c5)

for more detail please refer the Issue link: https://github.com/herlesupreeth/docker_open5gs/issues/259

if you have any idea please let me know.
Thank you in advance.
